jboss-eap-cd-openshift-launch/added/launch/activemq-subsystem.xml

```diff
@@ -1,6 +1,5 @@
     <server name="default">
         <statistics enabled="${wildfly.messaging-activemq.statistics-enabled:${wildfly.statistics-enabled:false}}" />
-        <journal pool-files="10"/>
```

Comment:
Does this module affect streams other than EAP CD?

Reply:
Used in 7.2

https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/eap72-dev/image.yaml#L60

It's in the 7.2 zip's standalone-full-ha.xml as well.

So this needs some research. I was involved in adding it and maybe with removing it but don't recall the details. Perhaps some missing default which required us to manually configure but was no longer necessary with 15. But might be needed in 7.2.

I'll check after dinner.

Reply:
This was why it came in....

https://github.com/wildfly/wildfly/pull/11665

Reply:
> Does this module affect streams other than EAP CD?

Yes, potentially, and thanks for the reminder. I'm going to version this module, as 7.2.0 is using it too.

Reply:
This is why it's not in CD 15:

https://github.com/wildfly/wildfly/commit/5dbfec513e19f012f87c43985b6d37b6f284af9e#diff-1fb640eb9732665bcc2d46253f3c7960

So, not needed because the management API of the subsystem sets the value if it's not configured in xml. But, in 7.2 that won't be the case. So, let's leave this in the xml. It does no harm, it's just redundant in CD 15.

Reply:
Added back, commit squashed.

Reply:
... and I forgot abotu the test again ;)